subgraph/src/mapping.ts

```diff
@@ -13,7 +13,7 @@ import {
 
 const txId = (ev: ethereum.Event, key: string): string => key + "-" + ev.logIndex.toString()
 
-const eventId = (ev: ethereum.Event): string => `${ev.transactionHash.toString()}-${ev.logIndex.toString()}`
+const eventId = (ev: ethereum.Event): string => `${ev.transaction.hash.toString()}-${ev.logIndex.toString()}`
```

Comment:
string interpolation is not supported currently AFAIK, concat strings with + for now